paddle/framework/scope.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <unordered_map>
+#include <vector>
+#include "paddle/framework/variable.h"
+#include "paddle/utils/Error.h"
```

Comment:
I don't see that this header file needs a special Error type.  I think if we follow below comments, we can remove this inclusion.